fasttext/yum/fasttext.spec.in

```diff
@@ -0,0 +1,80 @@
+Name:		fasttext
+Version:	0.9.1
+Release:	1%{?dist}
+Summary:	Efficient learning of word representations and sentence classification
+
+Group:		Applications/Text
+License:	MIT
+URL:		https://github.com/facebookresearch/fastText
+Source0:	fasttext-%{version}.tar.gz
+Patch0:		enable-pkg-config-for-0.9.1.patch
+
+BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-%(%{__id_u} -n)
+BuildRequires:	cmake3
+Requires:	%{name}-libs = %{version}-%{release}
+
+%description
+The fastText is a library for efficient learning of
+word representations and sentence classification.
+
+%package libs
+Summary:	Runtime libraries for fastText
+Group:		System Environment/Libraries
+Requires(post):	/sbin/ldconfig
+Requires(postun):	/sbin/ldconfig
+
+%description libs
+This package contains the libraries for fastText
+
+%package tools
+Summary:	Tools for fastText
+Group:		Applications/Text
+Requires:	%{name}-libs = %{version}-%{release}
+
+%description tools
+This package contains tools for manipulate models for fastText
+
+%package devel
+Summary:	Libraries and header files for fastText
+Group:		Development/Libraries
+Requires:	%{name}-libs = %{version}-%{release}
+
+%description devel
+This package contains header files to develop a software using fastText
+
+%prep
+%setup -q
+%patch0 -p1
+
+%build
+%cmake3 . -DCMAKE_INSTALL_LIBDIR=%{_libdir}
```

Comment:
`%cmak3` doesn't add `-DCMAKE_INSTALL_LIBDIR` automatically?

Reply:
Thanks, It's no need to specify explicitly. removed.